### DISCUSSION TITLE
Runtime error in Prototype-P8 configuration

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/WW3#773] [Q&A]
I am trying to run P8 on Frontera and am getting the following error:
Error termination. Backtrace:
At line 3223 of file /work2/02441/bcash/frontera/ufs-weather-model/Prototype-P8/WW3/model/src/w3updtmd.F90
Fortran runtime error: Index '0' of dimension 1 of array 'zz' below lower bound of 1

Is this an error people have seen before? I'm assuming it is because I have missed a file in moving my setup from Orion to Frontera but so far have had no luck figuring out what it is. Hoping this looks familiar to someone. :)

**Comment by JessicaMeixner-NOAA** [comment]
@benjamin-cash this does not look familiar to me.  Does your mod_def file that you created from a ww3_grid.inp file match what you are using for your wave mesh file in nems.configure?

**Reply by benjamin-cash** [reply]
Hi @JessicaMeixner-NOAA - I ran a couple of other tests over the weekend, and when I switched from gnu to intel it went away (??). All of the files you mentioned were generated via global workflow on Orion, and nems.configure is pointing to mesh.gwes_30m.nc, which is indeed present.

**Reply by JessicaMeixner-NOAA** [reply]
Glad to hear your issues went away!

**Reply by benjamin-cash** [reply]
Out of curiosity, is this the kind of difference you would expect in going from gnu to intel? Should it be possible to run using either compiler?

**Reply by JessicaMeixner-NOAA** [reply]
No this is not the difference I'd expect moving from GNU to intel, but again I've also never seen your error before.  Yes, I'd expect things to work with either.  I do know there might be issues with the latest versions of GNU (https://github.com/NOAA-EMC/WW3/issues/760), but these don't seem like those issues.
